### PR TITLE
Fix clang compilation error due to a fallthrough

### PR DESCRIPTION
**Pull request by Roman971**
Tested with clang 10.0.0 and 12.0.0

### CODE REVIEW

File: src/code/audio_effects.c [hunk 1]
```diff
@@ -243,7 +243,7 @@ f32 Audio_AdsrUpdate(AdsrState* adsr) {
             adsr->envIndex = 0;
             adsr->action.s.state = ADSR_STATE_LOOP;
             FALLTHROUGH;
-        retry:
+        retry:;
```
Comment (engineer124) [inline src/code/audio_effects.c:246]: Of course it's audio causing troubles again

**Review by engineer124 — APPROVED** [review]

**Review by EllipticEllipsis — APPROVED** [review]
Thanks, meant to do this myself but other stuff intervened.

**Review by fig02 — APPROVED** [review]

File: src/code/audio_effects.c (hunk 1 above)
Comment (Dragorn421) [reply src/code/audio_effects.c:246]: It's always the loudy neighbours being an impediment

**Review by Dragorn421 — APPROVED** [review]
Wait let me do a funny comment too

**Pull request merged**